core/chains/evm/config/v2/config.go

```diff
@@ -164,6 +169,10 @@ type EVMConfig struct {
 	Nodes EVMNodes
 }
 
+func (c *EVMConfig) IsEnabled() bool {
+	return c.Enabled == nil || *c.Enabled
```

Comment:
Here c.Enabled == nil indicates chain is enabled.
In other places, previous logic was mixed, some places treated this case as chain = disabled.

Not indicating any problem, just cautioning that tis PR is introducing a change in the chain enabled logic when the config was null